device/cuda/src/seeding/spacepoint_formation_algorithm.cu

```diff
@@ -60,12 +64,17 @@ spacepoint_formation_algorithm::operator()(
     const unsigned int nBlocks = (num_measurements + blockSize - 1) / blockSize;
 
     // Launch the spacepoint formation kernel.
-    kernels::form_spacepoints<<<nBlocks, blockSize, 0, stream>>>(
-        measurements, det_descr, num_measurements, spacepoints);
+    kernels::form_spacepoints<detector_t><<<nBlocks, blockSize, 0, stream>>>(
+        det_view, measurements_view, num_measurements, spacepoints);
     TRACCC_CUDA_ERROR_CHECK(cudaGetLastError());
 
     // Return the reconstructed spacepoints.
     return spacepoints;
 }
 
+// Explicit template instantiation
+template class spacepoint_formation_algorithm<default_detector::device>;
+template class spacepoint_formation_algorithm<telescope_detector::device>;
+template class spacepoint_formation_algorithm<toy_detector::device>;
```

Comment:
Ahh, I didn't even see this when I made my earlier comment. This is exactly why we should have explicit typedefs in the headers. To make it clear which specializations exist for which library...